etabotsite/default_settings.json

```diff
@@ -0,0 +1,45 @@
+{
+    "LOCAL_MODE": true,
+    "DOCKER_MODE": true,
+    "local_host_url":"http://127.0.0.1:8000",
+    "prod_host_url":"http://localhost:8000",
+    "AWS_SQS_REGION": "us-west-2",
+    "CELERY_DEFAULT_QUEUE": "etabotqueue",
+    "_MESSAGE_BROKER_comment":"rabbitmq or aws",
+    "MESSAGE_BROKER":"aws",
+    "RMQ_HOST" : "localhost",
+    "RMQ_VHOST" : "etabotvhost",
+    "db": {
```

Comment:
Please bring the default settings inline with the new naming convention for database. DB_NAME, DB_HOST, etc...

Reply:
done, please review and resolve if ok